src/js/common/browserInfoHelper.js

```diff
@@ -78,20 +90,65 @@ const getFormattedOffset = () => {
     return formattedUTC;
 }
 
+/**
+ * Function that returns user's timezone offset relative to UTC
+ * @returns {string} UTC concatenated with user's browser's timezone offset 
+ */
 export const getTimezone = () => `UTC${getFormattedOffset()}`;
+
+/**
+ * Function that validates {the user's screen's width value}, and then returns it.
```

Comment:
Hey, sorry, with the `{` and `}` I just meant that this part of the comment would change for each function - can you remove it from the comments please?

Reply:
@reubenae Done. Sorry for the confusion!